flexget/plugins/filter/series.py

```diff
@@ -240,6 +274,8 @@ class Series(Base):
     episodes = relation('Episode', backref='series', cascade='all, delete, delete-orphan',
                         primaryjoin='Series.id == Episode.series_id')
     in_tasks = relation('SeriesTask', backref=backref('series', uselist=False), cascade='all, delete, delete-orphan')
+    alternate_names = relation('AlternateNames', backref='series', primaryjoin='Series.id == AlternateNames.series_id',
```

Comment:
Pretty sure we don't need the primaryjoin argument since there is one ForeignKey linking these tables, it should be detected automatically.